src/utilities/time.ts

```diff
@@ -381,7 +381,7 @@ export function convertDoyToYmd(doyString: string, includeMsecs = true): string
       return `${ymdString.replace(/(\.\d+)/, '')}Z`;
     } else {
       // doyString is already in ymd format
-      return doyString;
+      return `${doyString}Z`;
```

Comment:
Were we always missing this timezone component of the date string here?

Reply:
I think so. We just didn't have anything fall through to this point.